api/tinynews-models/src/plugins/graphql/resolver.ts

```diff
@@ -35,6 +66,9 @@ export const resolveCreateFrom = (getModel: GetModelType): FieldResolver => asyn
   // ensure the new version shares the same parent, slug and env (what's the env? think we're missing that one)
   newRevision.parent = baseRevision.parent;
   newRevision.slug = baseRevision.slug;
+
+  newRevision.firstPublishedOn = baseRevision.firstPublishedOn;
+  newRevision.lastPublishedOn = baseRevision.lastPublishedOn;
```

Comment:
This will actually get overridden on new publishes, correct?

Reply:
Yeah - I need to write the `publishArticle` mutation that will update the dates (or more likely, date singular).